explore/src/main/scala/explore/Explore.scala

```diff
@@ -58,12 +60,10 @@ import js.annotation._
 @JSExportTopLevel("Explore")
 object ExploreMain extends IOApp.Simple {
 
-  implicit val reuseContext: Reusability[AppContextIO] = Reusability.never
-
   @JSExport
   def resetIOApp(): Unit =
     // https://github.com/typelevel/cats-effect/pull/2114#issue-687064738
-    cats.effect.unsafe.IORuntime.asInstanceOf[{ def resetGlobal(): Unit }].resetGlobal()
+    cats.effect.unsafe.JSPlatform.resetRuntime()
```

Comment:
@rpiaggio Did you test the reload with this?

Reply:
yup, seems to be working